docs/source/install-plone-restapi.rst

```diff
@@ -0,0 +1,77 @@
+Getting started with plone.restapi
+==================================
+If you haven't installed Plone locally, proceed to the next section, that is, installing Plone with plone.restapi. Make sure you have Python 2.7 and `virtualenv
+<https://virtualenv.pypa.io>`_ installed.
+If you have already set-up Plone locally, you can skip the next section and add plone.restapi as an add-on.
+
+Install Plone locally with plone.restapi
```

Comment:
@tulikavijay Isn't that rather the instructions for developing plone.restapi? I'd only recommend to checkout the plone.restapi repo if you want to work ON plone.restapi, not just build something on top of it.

Reply:
yes that might be the case. I think i should remove this section all-together and keep the next section. what do you think?